### PR TITLE
BZ1983533: Add URL to the table

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.8+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=1983533

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/add-url-firewall/installing/install_config/configuring-firewall.html

Additional information:

### CODE REVIEW

File: modules/configuring-firewall.adoc [hunk 1]
```diff
@@ -46,6 +46,10 @@ There are no special configuration considerations for services running on only c
 |443, 80
 |The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com`
 
+|`redhat-developer.github.io`
```
Comment (yapei) [inline modules/configuring-firewall.adoc:49]: I'm sorry, I didn't see any requests to `redhat-developer.github.io` when trying to browse Databases in Developer Console, here is my screenshot, could you please let me know how did you find this? 
it is sending requests to `https://<console_route>/api/devfile/samples/?registry=https://registry.devfile.io`
Screen Shot 2022-09-14 at 16 08 59

**Comment by xenolinux** [comment]
@yapei The changes should be valid for 4.8+ right?

File: modules/configuring-firewall.adoc [hunk 2]
```diff
@@ -46,6 +46,10 @@ There are no special configuration considerations for services running on only c
...
+|80, 443
+|
```
Comment (xenolinux) [inline modules/configuring-firewall.adoc:51]: @yapei would you please help me with the description of this parameter?

**Pull request closed without merging**